### PR TITLE
[OBSDEF-6531] support object store name `default`

### PR DESCRIPTION
**Pull request by ben-schumacher**
## Purpose
[OBSDEF-6531](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6531)
Added suffix "-storageserver" to service account, role and role binding.  This will allow an object store to be created with the name "default".

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
```
Charts:      26 passed, 26 total
Test Suites: 52 passed, 52 total
Tests:       250 passed, 250 total
Snapshot:    0 passed, 0 total
Time:        19.171734195s

```

### CODE REVIEW

**Review by spiegela — APPROVED** [review]
LGTM!

File: ecs-cluster/templates/ecs-cluster.yaml [hunk 1]
```diff
@@ -292,7 +292,7 @@ spec:
       debugLogging: {{ .Values.pravega.debugLogging }}
 
   storageServer:
-    serviceAccount: {{ .Release.Name }}
+    serviceAccount: {{ .Release.Name }}-storageserver
```
Comment (vgupta-mickey) [inline ecs-cluster/templates/ecs-cluster.yaml:295]: currently,  the service account is being used by the storage server. But do you think we might need to take the service account to other micro services?
if yes, we may need to create the account name: objectstore-{{ .Release.Name }}

**Review by vgupta-mickey — APPROVED** [review]
Please check my review comments.

File: ecs-cluster/templates/ecs-cluster.yaml (hunk 1 above)
Comment (ben-schumacher) [reply ecs-cluster/templates/ecs-cluster.yaml:295]: This is a good point, but everything is tested and working.  If you don't mind, I'd like to keep it this way and rename it in the future if it's used with other micro services.

**Pull request merged**